src/utils/utils.ts

```diff
@@ -4,7 +4,18 @@ import { SearchResponseModel } from "../models/dataset";
 import { FacetFilterModel } from "../models/facets";
 import { authService } from "../services/auth";
 
-const CLIENT_URL = process.env.REACT_APP_CLIENT_URL;
+
+/**
+ * Checks and cleans up URL if last character is or is not forward slash
+ * @param url - String of URL
+ * @param endSlash - whether we want to have an end slash or not
+ * @returns URL as specified
+ */
+export const cleanUrl = (url: string, endSlash : boolean = true) => {
+  const lastCharSlash : boolean = url.slice(-1) === "/"
+  return endSlash ? lastCharSlash ? url : url + "/" : lastCharSlash ? url.slice(0,-1) : url
+}
```

Comment:
I recommend using a dedicated function `withEndSlash()` because this conveys more clearly what the function does. And then, if needed (but I don't think we need it) have another function `withoutEndSlash()`. This will also make the function much simpler - easier to read and understand and less likely to have errors.

Boolean params are generally a code smell. For explanation, see [here](https://alexkondov.com/should-you-pass-boolean-to-functions/) for instance.

Reply:
Done 👍